.github/workflows/docker_release.yml

```diff
@@ -0,0 +1,56 @@
+name: Docker Release
+
+on:
+  push:
+    branches:
+      - release
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          release_branches: release
```

Comment:
Do you have documentation on how to manage releases and versions? Is this how you do it in the other projects?

Reply:
I will add the documentation